packages/sampler/stories/qa-stories/components/ScrollerResizableItem/ScrollerResizableItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@enact/moonstone/Button';
+
+
+class ScrollerResizableItem extends React.Component {
+
+	render () {
+		const height = this.props.more ? 1500 : 400;
+		const text = this.props.more ? 'less' : 'more';
```

Comment:
Please use `const` once.

Reply:
We can use `const` once, but I think it's recommended that using one const for one variable declaration from coding convention perspective. Most seem to be using it that way in enact libraries.

Reply:
It's ok if you think so.